examples/NSIDC/ISat2_from_CMR_to_S3.ipynb

```diff
@@ -0,0 +1,665 @@
+{
```

Comment:
Somewhere in this notebook, I think you need to include a link to documentation about the dataset you are searching for. I assume you have a dataset landing page somewhere.